client/my-sites/importer/importing-pane.jsx

```diff
@@ -123,6 +123,10 @@ class ImportingPane extends React.PureComponent {
 		);
 	};
 
+	getHeadingTextProcessing = () => {
+		return translate( 'Processing your file. It might take a while, please wait.' );
```

Comment:
As the next message will be something like "Upload Successful", should we chonge this to 'Uploading your file...'. Also it shouldn't take too long at all, so should we just say something along the lines of, "please wait a few moments"

Reply:
Personally I think "Processing your file" sounds OK in this context, even when the next message is "Success! File uploaded." I'd agree about the next bit though - "Please give us a few moments" or "Please wait a few moments" would probably sound friendlier.

Reply:
Thanks for the suggestions! Updated it to say "Processing your file. Please wait a few moments.". We can update it later if we need to.